app/uk/gov/hmrc/tariffclassificationfrontend/views/case_summary.scala.html

```diff
@@ -30,7 +30,7 @@ <h1 class="heading-large">#@c.reference - @c.application.getType() for @c.applic
             @case_nav("summary", c.reference)
         </div>
         <div class="column-three-quarters">
-            <h3 class="heading-medium">Summary</h3>
+            <h3 class="heading-medium" style="margin-top: 0">Summary</h3>
```

Comment:
Could we group the styles on the css? :)

Reply:
Sorted